akka-docs/src/test/java/jdocs/stream/operators/SourceOrFlow.java

```diff
@@ -271,6 +273,23 @@ void groupedExample() {
     // #grouped
   }
 
+  CompletionStage<Integer> asyncFunction(int acc, int next) {
+    return CompletableFuture.supplyAsync(() -> acc + next);
+  }
+
+  void scanAsyncExample() {
+    // #scan-async
+    Source<Integer, NotUsed> source = Source.range(1, 5);
+    source.scanAsync(0, (acc, x) -> asyncFunction(acc, x)).runForeach(System.out::println, system);
```

Comment:
I think it would be more clear if the `asyncFunction` somehow was included in the snippet so one can see the CompletionStage signature

Reply:
Yep, meant to do that but forgot to add the tags